editor/plugins/canvas_item_editor_plugin.h

```diff
@@ -633,6 +633,7 @@ class CanvasItemEditorViewport : public Control {
 	Label *label_desc = nullptr;
 	VBoxContainer *btn_group = nullptr;
 	Ref<ButtonGroup> button_group;
+	EditorSelection *editor_selection = nullptr;
```

Comment:
This isn't really necessary. You can fetch it on spot, like EditorUndoRedoManager.

Reply:
Thanks, I originally did it since it's used in the same file in the other parts too and I think even in the 3D one but now I updated it as you suggested, better this way?